File: works/tmp/5543.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
    int a,b,c,d,e; cin >> a >> b >> c >> d >> e;
    int res1 = min(a, min(b,c));
    int res2 = min(d,e);
    cout << res1 + res2 - 50;
}

File: works/js/4-1/5543.js
let fs = require('fs');
let input = fs.readFileSync(0).toString().trim().split('\n');
// let input = fs.readFileSync('input.txt').toString().split('\n');

let a = parseInt(input[0]), b = parseInt(input[1]), c = parseInt(input[2]);
let d = parseInt(input[3]), e = parseInt(input[4])

let res1 = Math.min(a, Math.min(b,c))
let res2 = Math.min(d, e)
console.log(res1 + res2 - 50)